test/util.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/bash
```

Comment:
Hi Zephraph,

I am sorry for all this messed PR. I planned (again) to squash it and leave
out the "keyword_press_keys" branch. I failed to have it cleaned up :(.

The main differences in my fork are:
- I use RIDE so, most of the test cases will have 4 spaces and headings
like "*** Settings ***" as opposed to "* Setting *"
- My goal is to have Selenium2Library tests passing OK with all browsers
(so far tested with Firefox, Chrome, Opera, PhantomJS in Linux, and IE in
Windows).

Since I started using the .robot extension, I could not have my fork merged
with the original, that is why I was trying to get this solved.

Please do what you feel best, cherry-pick if you can spare that effort or
simply reject the PR.

Sorry for all this work (I am still new to Git ;) ).

On Mon, Jul 6, 2015 at 4:34 AM, Zephraph <notifications@github.com> wrote:

> In test/util.sh
> <https://github.com/rtomac/robotframework-selenium2library/pull/440#discussion_r33904229>
> :
>
> > @@ -0,0 +1,29 @@
> > +#!/usr/bin/bash
>
> Heh, I wish I would have had this script a week or two ago! I actually
> wrote my own to correct all these references (see 2456984
> <https://github.com/rtomac/robotframework-selenium2library/commit/245698483ff52f6e9512a9314ff13c49c29f37a8>).
> I'm not opposed to adding this to correct any future mistakes, but I don't
> think it should be done in this PR. Mind moving this script out into its
> own PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtomac/robotframework-selenium2library/pull/440/files#r33904229>
> .
>